src/m365/spo/commands/group/group-member-add.ts

```diff
@@ -171,11 +178,11 @@ class SpoGroupMemberAddCommand extends SpoCommand {
 
     const validUserNames: string[] = [];
     const invalidUserNames: string[] = [];
-    const userInfo: string = args.options.userName ? args.options.userName : args.options.email!;
+    const userInfo: string = args.options.userName ? args.options.userName : args.options.email ? args.options.email : args.options.userId!.toString();
```

Comment:
Can't we just write `args.options.userName || args.options.email || args.options.userId`?